index.html

```diff
@@ -40,155 +50,224 @@ <h5 class="subtitle">I'm a Softwere Developer</h5>
+            </article>
+            <article class="technology">
+                <ul>
+                    <li>
+                        <h3 class="skills">Bootstrap</h3>
+                    </li>
+                    <li>
+                        <h3 class="skills">Ruby on Rails</h3>
+                    </li>
+                    <li>
+                        <h3 class="skills">RSPec</h3>
+                    </li>
+                    <li>
+                        <h3 class="skills">CapyBara</h3>
+                    </li>
+                    <li>
+                        <h3 class="skills">Selenium</h3>
+                    </li>
+                </ul>
+            </article>
+            <article class="tools">
+                <div class="circle"></div>
+                <h6>Skills</h6>
+            </article>
+            <article class="toolist">
+                <ul>
+                    <li>
+                        <h3 class="skills">Codekit</h3>
+                    </li>
+                    <li>
+                        <h3 class="skills">Github</h3>
+                    </li>
+                    <li>
+                        <h3 class="skills">Codepen</h3>
+                    </li>
+                    <li>
+                        <h3 class="skills">Gitlab</h3>
+                    </li>
+                    <li>
+                        <h3 class="skills">Terminal</h3>
+                    </li>
                 </ul>
             </article>
-                     <article class="tools">
-                            <div class="circle"></div>
-                            <h6>Skills</h6>
-                        </article>
-                    <article class="toolist">    
-                        <ul>
-                            <li><h3 class="skills">Codekit</h3></li>
-                            <li><h3 class="skills">Github</h3></li>
-                            <li><h3 class="skills">Codepen</h3></li>
-                            <li><h3 class="skills">Gitlab</h3></li>
-                            <li><h3 class="skills">Terminal</h3></li>
-                        </ul>
-                    </article>
+        </section>
+        <section class="form-section">
+            <article class="form-container">
+                <h2 class="form-title">
+                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+                </h2>
+                <form action="https://formspree.io/f/xyyoqkjw" method="post" class="contact-form">
+                    <input type="text" name="full-name" maxlength="30" placeholder="Full Name" required>
+                    <input type="email" name="email" placeholder="Email address" required>
+                    <textarea name="message"  cols="30" rows="10" placeholder="Enter text here" required maxlength="500"></textarea>
+                    <button type="submit" class="btn-contact">Get in touch</button>
```

Comment:
- [ ] You have done a tremendous job styling your submit button. However, so that you stick as much as possible to the chosen Figma design, kindly make sure that you implement the button states eg on hover, on submit etc.This is the missing piece to perfection.

**Below are the desired button states**
![image](https://user-images.githubusercontent.com/86318284/177398602-f1616c41-9c7c-414b-95c4-b76238f47f9b.png)

- [ ] Kindly make sure that your submit button looks like the chosen Figma design. 

**This is what your design currently looks like:**
![image](https://user-images.githubusercontent.com/86318284/177399942-14221101-e23d-4adc-8584-cea930eee201.png)

**This is the desired Figma design:**
![image](https://user-images.githubusercontent.com/86318284/177400072-2e7d66c9-2766-4024-b646-7971b8473059.png)